source/v4/teams.yaml

```diff
@@ -69,6 +69,35 @@
         '404':
           $ref: '#/responses/NotFound'
 
+  '/teams/name/{name}':
+    get:
+      tags:
+        - teams
+      summary: Get a team based on name string query parameter
```

Comment:
Can we shorten this to `Get team by name`? This will end up in a sidebar so it's nice to keep it short